apps/server/src/modules/board/controller/api-test/content-element-create.api.spec.ts

```diff
@@ -28,11 +28,11 @@ class API {
 		this.app = app;
 	}
 
-	async post(cardId: string, type: ContentElementType) {
+	async post(cardId: string, type: ContentElementType, toPosition?: number) {
```

Comment:
the whole class can be removed - not needed
instead, call directly TestApiClient
also, for the users, instead of calling the factory in setup, include and use UserAndAccountTestFactory
example in board-update-title.api.spec.ts

Reply:
refactoring of specs in a followUp-Ticket?